event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/JdbcEventRepositoryTest.java

```diff
@@ -95,6 +102,79 @@ public void shouldGetAll() throws Exception {
         verify(logger).trace("Retrieving all events");
     }
 
+    @Test
+    public void shouldGetStreamOfStreams() throws Exception {
+        final UUID streamId1 = UUID.fromString("4b4e80a0-76f7-476c-b75b-527e38fb251e");
+        final UUID streamId2 = UUID.fromString("4b4e80a0-76f7-476c-b75b-527e38fb252e");
+        final UUID streamId3 = UUID.fromString("4b4e80a0-76f7-476c-b75b-527e38fb253e");
+
+        final EventLog event1 = new EventLog(null, streamId1, null, null, null, null, null);
```

Comment:
Could we move some of this into a private creator as it is reused ?
eventLogOf(steadId1) etc ? - similar to below test.